Kconfig

```diff
@@ -1,4 +1,28 @@
 menu "BMX280 Options"
+    choice I2C_DRIVER_SETTING
+        prompt "I2C driver setting"
+        help
+            Select I2C driver: I2C_OLD: Use old driver I2C.h, I2C_NEW: Use new i2c_master.h(ESP-IDF >=5.3)
+        default USE_I2C_OLD_DEVICE
```

Comment:
I don't like the new terminology you have introduced here. New/Old might imply that the BMP280/BME280 devices being used are new and old.

I suggest you rename these config keys and their help strings as I2C Master and I2C Legacy.